app/agent/activities/archive-prebuild-event/delete-local-prebuild-event-files.ts

```diff
@@ -12,26 +13,27 @@ export const deleteLocalPrebuildEventFiles: CreateActivity<
   DeleteLocalPrebuildEventFilesActivity
 > = ({ injector, db }) =>
   withActivityHeartbeat({ intervalMs: 1000 }, async (args) => {
-    const prebuildService = injector.resolve(Token.PrebuildService);
+    const brService = injector.resolve(Token.BlockRegistryService);
     const prebuildEvent = await db.prebuildEvent.findUniqueOrThrow({
       where: { id: args.prebuildEventId },
       include: {
-        prebuildEventFiles: {
+        prebuildEventImages: {
           include: {
             agentInstance: true,
-            fsFile: true,
-            projectFile: true,
+            fsImage: true,
+            projectImage: true,
           },
         },
       },
     });
-    const files = unwrap(
-      prebuildEvent.prebuildEventFiles.find(
+    const images = unwrap(
+      prebuildEvent.prebuildEventImages.find(
         (f) => f.agentInstance.externalId === SOLO_AGENT_INSTANCE_ID,
       ),
     );
-    await prebuildService.removeLocalPrebuildEventFiles({
```

Comment:
I don't see an activity for an periodic garbage collection. Do you want to run GC here?

Reply:
GC is actually run after every prebuild in https://github.com/hocus-dev/hocus/blob/HOC-197-fetchrepo-overlaybd/app/agent/activities/registry/remove-content-with-prefix.ts#L14